experimental/framework/frs-client/README.md

```diff
@@ -6,16 +6,37 @@ This package is marked as experimental and currently under development. The API
 
 ## Using frs-client
 
-The frs-client package has a `FrsClient` static class that allows you to interact with Fluid.
+The frs-client package has a `FrsClient` class that allows you to interact with Fluid.
 
 ```javascript
 import { FrsClient } from "@fluid-experimental/frs-client";
 ```
 
-## Initializing FrsClient
+## Instantiating FrsClient
 
-Fluid requires a backing service to enable collaborative communication. The FrsClient instance will be instantiated against the FRS service.
+Fluid requires a backing service to enable collaborative communication. The `FrsClient` supports both instantiating against a deployed FRS service instance for production scenarios, as well as against a local, in-memory service instance, known as Tinylicious, for development purposes.
 
+NOTE: You can use one instance of the `FrsClient` to create/fetch multiple containers from the same FRS service instance.
+
+### Backed Locally
+
+When running against Tinylicious, the `FrsClient` constructor just takes in two optional parameters, the domain and port that Tinylicious is running at. To run Tinylicious on the default values of `localhost:7070`, please enter the following into a terminal window:
+```
+npx tinylicous
+```
+Now, with our local service running in the background, we need to connect the application to it. For this, we just need to specify that we are running in local mode when instantiating our `FrsClient`
+
+```javascript
```

Comment:
nit: these should be ` ```typescript` right? though it doesn't look like it matters for most of them

Reply:
well nvm, looks like the md does the right thing anyway 🤔

Reply:
Updated to be typescript jic 👍